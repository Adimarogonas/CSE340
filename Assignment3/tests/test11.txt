{
    while (+ < r s true) {
        r = * r s;
    }
    while (* > t u < v w) {
        t = + t w;
    }
}
