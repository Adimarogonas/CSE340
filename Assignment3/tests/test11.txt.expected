r, s: ? #
t, u, v, w: ? #
